Mobile Version

Malcom-Charlie opened the pull request:
> These are the Project requirements that have been met:
 
## General Requirements:
- Ensure no linter errors.
- Use the correct Github flow.
- Professional documentation.

## HTML/CSS requiments:
- Follow list of best practises

## Project Requirements:
- Build the middle section of the mobile website.

-    Work section:
![works-mobile](https://user-images.githubusercontent.com/61559987/171967730-40423222-5941-4996-b3b6-50cc81d6c1fb.png)

-   About myself 
<img width="188" alt="about-myself-mobile" src="https://user-images.githubusercontent.com/61559987/171967824-b57f7e0e-d501-4b1c-9942-efbc83adfd3d.png">

-  Use  CSS Grid and Flexbox combined.
-  Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.
-   Use `ul` and `li` for categories in the "Work" section. `Html`, `CSS`, and `Javascript` should be list items.
-  Use `ul` and `li` for categories in the "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items.
-  No need to implement the detail modal window that requires JavaScript.
-  Stick to the design as much as possible.
-  Implement the button interactions (enable, hover, etc.).

Malcom-Charlie committed:
Finished work section cards

Malcom-Charlie committed:
Finished building About section

Malcom-Charlie committed:
Intergrated hover functionality

Malcom-Charlie committed:
Added the grid layout sction

Malcom-Charlie committed:
Updated README.md file

Malcom-Charlie committed:
Corrected Webhint Errors and warnings

I only corrected errors and warnings left out the hints as they interfere with code

Malcom-Charlie committed:
Applied Stylelint changes

Malcom-Charlie committed:
Applied 2nd Stylelint changes

Kakalanp commented on index.html:
```diff
@@ -6,35 +6,187 @@
+  <section class="intro">   
+    <div class="space"></div>
+      <h1 class="dcp1 art1">
+      I’m Sir_Malcom<br>Glad to see<br>you!
```
[OPTIONAL]

I highly encourage you to use your real name instead of a surename, it will definetly make your portfolio seem more professional 💼

Kakalanp commented on style.css:
```diff
@@ -9,59 +9,61 @@
-.parent1 {
+.intro {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  width: 376px;
-  height: 100vh;
+  padding: 16px;
+  background: #fff;
+  border-radius: 16px;
+  max-width: 327px;
+  margin: auto;
+  margin-bottom: 70px;
   background-image: url(./elements/bglmnt.png);
   background-repeat: no-repeat;
 }
```
Thanks to padding and margins your background seems a little off, you can't bypass margins, but you can ignore padding in your background-image by adding the property and value: `box-sizing: content-box;`, kindly add that line to this class to make it look similar to the Figma design 💯

Kakalanp commented on index.html:
```diff
@@ -6,35 +6,187 @@
+      <img src="./elements/snpsht.png" alt="Tonic">
+      <div class="prmry-txt">
+            <h2 class="title">Tonic</h2>
+            <div class="frm-2">
+              <span class="clnt">CANOPY</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="role">Back End Dev</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="year">2015</span>
+            </div>
```
[OPTIONAL]

I inspire you to use a `<ul>` tag and `<li>`'s for this section of the projects, although it is not a requirement for this project, it is considered good practice, something that surely will stand out when somebody sees your portfolio 😉

Malcom-Charlie replied:
@Kakalanp I am unable to implement this suggestion right now due to time constraints. However, I have noted it and will endeavor to do it in the manner suggested

Kakalanp commented on index.html:
```diff
@@ -6,35 +6,187 @@
+  </section>
+  <section class="about">
+    <section class="abtmyslf art1">
+      About<br>Myself
```
Remember that this sub**title**, so this must use an: <details><summary> `<hx>`</summary>
    <pre>
    - `<h2>`
    - `<h3>`
    - `<h4>`
    - `<h5>`
    </pre>
   </details> to accomplish this 🔥

Kakalanp commented on index.html:
```diff
@@ -6,35 +6,187 @@
+      If you like what you see and have a project you need coded,
+      don’t hesitate to contact me.
+    </p>
+    <p class="ltscnct">LET’S CONNECT</p>
```
Not to be repetitive, but this is also a kind of subtitle, so kindly try to use an: <details><summary> `<hx>`</summary>
    <pre>
    - `<h3>`
    - `<h4>`
    - `<h5>`
    </pre>
   </details> for this 😃

Kandy-Peter replied:
- [ ] For the same reasons, it would be nice if you implement the same HTML subtitle `hx` tag here.

Kakalanp reviewed (changes requested):
# Hello there @Malcom-Charlie! <img src="https://user-images.githubusercontent.com/1303154/88677602-1635ba80-d120-11ea-84d8-d263ba5fc3c0.gif" width="28" alt="hi">

### Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

<img src="https://media.giphy.com/media/Qaxd5XpgDWMso/giphy.gif" alt="so close">

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Very organized code :monocle_face: 
- Excellent UI design 🥇 

## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Malcom-Charlie committed:
Implemented code review changes

Malcom-Charlie committed:
Implemented code review changes

Kandy-Peter commented on index.html:
```diff
@@ -6,35 +6,187 @@
+  </section>
+  <section class="about">
+    <section class="abtmyslf art1">
+      About<br>Myself
```
- [ ] Please use a `hx` tag(h2, h3, h4...) as the `Ab0ut me` is a subtitle. We suggest this because the heading tags can provide both visitors and the search engines with significant cues to the importance of your on-page content.

Kandy-Peter reviewed (changes requested):
Hi @Malcom-Charlie,

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some issues which aren't been addressed yet.

![image](https://user-images.githubusercontent.com/80612925/169586832-56418ba8-59ea-4376-97ab-65fb0f51c869.png)

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)..

Malcom-Charlie committed:
Implemented code review changes

Malcom-Charlie committed:
Implemented indirect code review chnge 

A change in the Html as required by code review necessitated a change in style sheet

Cathella commented on index.html:
```diff
@@ -139,16 +139,16 @@ <h2 class="title">Tonic</h2>
   </section>
   <section class="about">
-    <section class="abtmyslf art1">
+    <h1 class="abtmyslf art1">
```
- [ ] [OPTIONAL] Kindly make this an `h2`, `h3`, `hx`, etc but not an `h1` because we already have one in the same page.

Cathella reviewed (approved):
## 🔥 Status: Approved 🟢

### Hi @Malcom-Charlie 👋🏻, 

Good job on making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

![rosycheeks-mochi-peach](https://user-images.githubusercontent.com/4247740/171981129-52a58594-3d9e-4b9f-8c10-0a5a221c0cac.gif)


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

roshan-bajgain reviewed (commented):
Hi @Malcom-Charlie, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a different link. So kindly see the project as submit the right one

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

EricMbouwe reviewed (commented):
Hi @Malcom-Charlie , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
